charts/flyte-core/values-eks.yaml

```diff
@@ -326,6 +325,7 @@ configmap:
 # this module, you will not have scheduled launchplans/workflows.
 workflow_scheduler:
   enabled: false
+  type: native
```

Comment:
Why native in values-eks. 
We want this to be native in sandbox  gcp and other cloud providers except aws